README.md

```diff
@@ -64,12 +48,10 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
```

Comment:
Please give appropriate Credit to [Cindy Shin] in "Acknowledgements" section (https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).